R/my_first_fcn.R

```diff
@@ -19,6 +19,8 @@
 #' @examples
 #' hello_admiral(hw = FALSE)
 hello_admiral <- function(hw = TRUE) {
```

Comment:
Not sure if we intend to merge this dummy function into devel. In general, please follow the function naming convention and programming strategy as outline in the [programming stragety guide](https://pharmaverse.github.io/admiraldev/main/articles/programming_strategy.html)

Reply:
Thanks Annie.  Going to remove once we get a few functions into this package